tests/Unit/Scraper/Listeners/ConfigureScraperTest.php

```diff
@@ -209,13 +209,14 @@ public function whenTheIdStoreIsNotAvailableItShouldThrowAnUnexpectedValueExcept
         Log::shouldReceive('debug');
         Log::shouldReceive('error')
             ->with("Error scraping ':scrape-url:'", ['message' => ':error:']);
-        $this->expectsEvents(ScrapeFailed::class);
 
+            
```

Comment:
Avoid double empty lines and Space the Event::assertDispatched to let the code breathe there, check other tests. I see that I need to add an improved checkstyle :pray:.